### PR TITLE
Add docker image and pipeline_config.json

### PR DESCRIPTION
**Pull request by IsaackMwenda**

### CODE REVIEW

File: configuration/docker_image_project_name.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+imaqal_covid19
```
Comment (as2388) [inline configuration/docker_image_project_name.txt]:
```suggestion
imaqal-covid19
```
for consistency with the other names.
I _think_ the naming convention for Docker is lower case and hyphen separated

File: configuration/pipeline_config.json [hunk 2]
```diff
@@ -0,0 +1,69 @@
+{
+  "PipelineName": "IMAQAL_COVID19",
+  "RawDataSources": [
+    {
+      "SourceType": "RapidPro",
+      "Domain": "textit.in",
+      "TokenFileURL": "gs://avf-credentials/imaqal-text-it-token.txt",
+      "ContactsFileName": "imaqal_covid19_contacts",
+      "ActivationFlowNames": [
+        "imaqal_mag_c19_s01e01_activation"
```
Comment (as2388) [inline configuration/pipeline_config.json]: If it's not too late, can this be covid19 instead of c19?

**Review by as2388 — APPROVED** [review]

File: configuration/docker_image_project_name.txt (hunk 1 above)
Comment (IsaackMwenda) [reply configuration/docker_image_project_name.txt]: updated thanks

File: configuration/pipeline_config.json (hunk 2 above)
Comment (IsaackMwenda) [reply configuration/pipeline_config.json]: We are still in dev mode. updated to imaqal_covid19_mag_s01e01_activation. Thanks.

**Pull request merged**